### PR TITLE
PUBDEV-7060: Propagate offset_column to MOJO models

### PR DESCRIPTION
**Pull request by michalkurka**

### CODE REVIEW

File: h2o-genmodel/src/main/java/hex/genmodel/AbstractMojoWriter.java [hunk 1]
```diff
@@ -180,6 +180,9 @@ private void addCommonModelInfo() throws IOException {
     writekv("n_classes", model.nclasses());
     writekv("n_columns", model.columnNames().length);
     writekv("n_domains", n_categoricals);
+    if (model.offsetColumn() != null) {
```
Comment (Pscheidl) [inline h2o-genmodel/src/main/java/hex/genmodel/AbstractMojoWriter.java]: Having it for all makes more sense. Great.

File: h2o-genmodel/src/main/java/hex/genmodel/GenModel.java [hunk 2]
```diff
@@ -112,6 +112,10 @@ public GenModel(String[] names, String[][] domains) {
     return _domains.length - 1;
   }
 
+  @Override public String getOffsetName() {
```
Comment (Pscheidl) [inline h2o-genmodel/src/main/java/hex/genmodel/GenModel.java]: getOffsetColName ? Doesn't matter really, the var is named `_offsetColumn`, not offset.

Comment (michalkurka) [reply h2o-genmodel/src/main/java/hex/genmodel/GenModel.java]: this is modeled to be consistent with `getResponseName`

File: h2o-genmodel/src/main/java/hex/genmodel/AbstractMojoWriter.java (hunk 1 above)
Comment (michalkurka) [reply h2o-genmodel/src/main/java/hex/genmodel/AbstractMojoWriter.java]: thank you

**Pull request merged**